yarn.lock

```diff
@@ -1545,6 +1545,32 @@
   resolved "https://registry.yarnpkg.com/@fluentui-react-native/design-tokens-windows/-/design-tokens-windows-0.14.0.tgz#1483caa1dd4e7097ac48a5872e93d7f896e8402c"
   integrity sha512-WNNVTMpRir6a2Voa0yu3BZJmy2w7fofrokL+nyFQM40bACtX68KDRDE96HA5W5+4q8rEaNSXV3DzSzLXm6ekfA==
 
+"@fluentui-react-native/experimental-shadow@0.2.12":
+  version "0.2.12"
+  resolved "https://registry.yarnpkg.com/@fluentui-react-native/experimental-shadow/-/experimental-shadow-0.2.12.tgz#79b82ca1eb1c4ef346deb99bd4729df3d8d04a7d"
+  integrity sha512-g4rcHAUN6mjRTedgaW9OqUsU5bYkVPG1alr/kPH5FKOvkld/42/+ZjVDg8yl0qxYUK+/8BO7floLbGySrwhlpQ==
+  dependencies:
+    "@fluentui-react-native/framework" "0.8.0"
+    "@fluentui-react-native/theme-types" ">=0.18.0 <1.0.0"
+
+"@fluentui-react-native/framework@0.8.0":
```

Comment:
framework is now on a later version, please update your package.json

Reply:
That's strange, because I have correct version  `"@fluentui-react-native/framework": ">=0.8.3 <1.0.0"`

Reply:
Odd. I wonder where it's coming from then